CMakeLists.txt

```
@@ -2098,8 +2119,8 @@ target_link_libraries(mixxx-lib PUBLIC
   Qt5::Widgets
   Qt5::Xml)
 target_compile_definitions(mixxx-lib PUBLIC QT_TABLET_SUPPORT QT_USE_QSTRINGBUILDER)
-get_target_property(QT5_TYPE Qt5::Core TYPE)
-if(QT5_TYPE STREQUAL "STATIC_LIBRARY")
+is_static_library(Qt5_IS_STATIC Qt5::Core)
```

```suggestion
is_static_library(Qt_IS_STATIC Qt5::Core)
```
to avoid needing to rename this when adding Qt6 support

---

Qt5 reflects the target name as everywhere else. I like to keep it consistent.

---

okay, I'll rename this when rebasing #4052